okapi-common/src/main/java/org/folio/okapi/common/ErrorType.java

```diff
@@ -4,39 +4,36 @@
  * Types of errors.
  */
 public enum ErrorType {
-  /** Not really an error, but a success code */
+  /** Not really an error, but a success code. */
   OK,
-  /** Internal errors of any kind */
+  /** Internal errors of any kind. */
   INTERNAL,
   /** Bad requests, etc. */
   USER,
-  /** Stuff that is not there */
+  /** Stuff that is not there. */
   NOT_FOUND,
-  /** Any kind of auth or permission problem */
+  /** Any kind of auth or permission problem. */
   FORBIDDEN,
-  /** Error type for anything else */
+  /** Error type for anything else. */
   ANY;
 
+  /**
+   * Maps error type to HTTP status code.
+   * @param t error type
+   * @return HTTP status
+   */
   public static int httpCode(ErrorType t) {
-    int code = 500;
     switch (t) {
       case OK:
-        code = 200;
-        break;
+        return 200;
       case USER:
-        code = 400;
-        break;
+        return 400;
       case NOT_FOUND:
-        code = 404;
-        break;
+        return 404;
       case FORBIDDEN:
-        code = 403;
-        break;
-      case INTERNAL:
-      case ANY:
-        code = 500;
-        break;
+        return 403;
+      default:
+        return 500;
```

Comment:
This unnecessary default clause prevents detection of missing enum values if we add more in future.
https://google.github.io/styleguide/javaguide.html#s4.8.4.3-switch-default
Sonar checks for enum completeness: https://sonarcloud.io/organizations/folio-org/rules?open=java%3AS131&rule_key=java%3AS131

Reply:
I got a warning, because no default was added. Pedantic

Reply:
You may avoid the switch by using an httpCode variable like in the Planet example
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html